rest/test_results_routes.go

```diff
@@ -14,20 +14,15 @@ import (
 	"github.com/pkg/errors"
 )
 
-const (
-	testResultsStartAt = "started_after"
-	testResultsEndAt   = "finished_before"
-)
-
 ///////////////////////////////////////////////////////////////////////////////
 //
 // GET /testresults/task_id/{task_id}
 
 type testResultsGetByTaskIdHandler struct {
-	taskId   string
-	interval model.TimeRange
-	tags     []string
-	sc       data.Connector
+	taskId   		string
```

Comment:
a little nit, but instead of having these options here you can just have a field `options` of type `model.TestResultsFindOptions` and populate it in the `Parse` method.